### PR TITLE
Updating to Naut CSS 

### PR DESCRIPTION
**Pull request by LinearArray**
Moving to Naut for easier maintenance and accessibility. Put logo as sidebarimg and banner has headerimg. Sidebar markdown code is in sidebar.md

### CODE REVIEW

**Comment by LinearArray** [comment]
Also just clarifying banners needs to be uploaded as `headerimg` and logo needs to be uploaded as `sidebarimg` in the stylesheet. 

Sidebar markdown code should be pasted in `subreddit settings > sidebar`.

**Comment by prakhar-garg-dev** [comment]
@Linear-Array Can you post the main link from where you are referring naut css from

File: src/main.css [hunk 1]
```diff
@@ -1,99 +1,862 @@
...
+}
+
+/* Sidebar button for twitter. */
+.side .md a[href="https://twitter.com/devsinindia"] {
+	display: block;
+	background-color: #ffffff;
+	color: #2CA5E0;
+	line-height: 3ex;
+	padding: 7px 7px 7px 40px;
+	background-image: url(%%icons8-twitter-40%%);
```
Comment (Bhupesh-V) [inline src/main.css]: Can we align the name of the file with the image name here?

File: src/main.css [hunk 2]
```diff
@@ -1,99 +1,862 @@
...
+}
+
+/* Sidebar button for Discord. */
+.side .md a[href="https://discord.gg/developersindia-669880381649977354"] {
+	display: block;
+	background-color: #7289DA;
+	color: #FFF;
+	line-height: 3ex;
+	padding: 7px 7px 7px 45px;
+	background-image: url(%%rsz-discord-logo-300x300%%);
```
Comment (Bhupesh-V) [inline src/main.css]: Can we align the name of the file with the image name here?

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]
Have updated the old sidebar, added 2 comments for review

File: src/main.css (hunk 2 above)
Comment (LinearArray) [reply src/main.css]: You'll need to set a custom image name when upload it.

File: src/main.css [hunk 3]
```diff
@@ -1,99 +1,864 @@
...
+
+    #header-bottom-right a:hover {
+        text-decoration: underline;
+    }
+
+.beta-hint a {
+    margin-left: -16px;
+    width: 16px;
+    height: 16px;
+    background: url(%%spritesheet%%) -48px -32px no-repeat;
```
Comment (Bhupesh-V) [inline src/main.css:297]: What is spritesheet image? Please add that in PR

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]

**Comment by LinearArray** [comment]
Done, added.

File: src/main.css [hunk 4]
```diff
@@ -114,130 +879,3625 @@ a.author.moderator, .res a.author.moderator {
...
+#eu-cookie-policy .c-btn-primary {
+    margin: -2px 16px 0 0;
+}
+
+/* No Results */
+#noresults {
+    margin: 0;
+    width: 100%;
+    height: 416px;
+    background: transparent url(%%noresults%%) no-repeat 50% 50%;
```
Comment (Bhupesh-V) [inline src/main.css:1895]: Missing image

File: src/main.css [hunk 5]
```diff
@@ -114,130 +879,3625 @@ a.author.moderator, .res a.author.moderator {
...
+}
+
+    .commentarea .usertext-edit textarea {
+        position: relative;
+        z-index: 0;
+        width: 100%;
+    }
+
+.commentarea > .usertext textarea:not(:focus) {
+    background: url(%%commentboxbg%%) no-repeat 50% 50%;
```
Comment (Bhupesh-V) [inline src/main.css:2617]: missing image

File: src/main.css [hunk 6]
```diff
@@ -114,130 +879,3625 @@ a.author.moderator, .res a.author.moderator {
...
+        font-size: 14px;
+        margin-left: 8px;
+    }
+
+/* Footer */
+.footer-parent {
+    height: 800px;
+    margin-top: 192px;
+    padding: 528px 0 64px;
+    background: url(%%footer%%) 50% 0 / cover no-repeat;
```
Comment (Bhupesh-V) [inline src/main.css:3849]: missing image

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]

**Comment by LinearArray** [comment]
done

**Pull request merged**